Makefile

```diff
@@ -46,7 +46,7 @@ RELEASE_OUT ?=
 TELEPORT_TAG = 3.2.14
 # TELEPORT_REPOTAG adapts TELEPORT_TAG to the teleport tagging scheme
 TELEPORT_REPOTAG := v$(TELEPORT_TAG)
-PLANET_TAG := 7.0.15-$(K8S_VER_SUFFIX)
+PLANET_TAG := 7.0.16-11602-3-g59c77a2
```

Comment:
yeah, it's confusing ... that tag is based on the old one, thus seems bound to a different version, while it's actually using the newer version.
Using the current Jenkins' job based workflow, we're bound to have this hiccup.
Or at least so far I haven't found a way to workaround it.

Maybe @r0mant knows a different way to use the "future" target tag instead of the old one.

Reply:
No - it's fine if it was the new one indeed.